rs/rosetta-api/icp_ledger/tests/upgrade_downgrade.rs

```diff
@@ -400,6 +400,7 @@ fn should_set_up_initial_state_with_mainnet_canisters() {
     assert!(ledger_archives.is_empty());
 }
 
+#[ignore]
```

Comment:
can you explain why this test is ignored?

Reply:
@nmattia: Yes, it's related to #1002. We have [FI-1330](https://dfinity.atlassian.net/browse/FI-1330) to fix this permanently, but unfortunately it hasn't had high enough priority to get addressed yet.